pkg/controller/certificates/cleaner/cleaner.go

```diff
@@ -172,7 +172,7 @@ func isIssuedPastDeadline(csr *capi.CertificateSigningRequest) bool {
 
 // isOlderThan checks that t is a non-zero time after time.Now() + d.
 func isOlderThan(t metav1.Time, d time.Duration) bool {
-	return !t.IsZero() && t.Sub(time.Now()) < -1*d
+	return (!t.IsZero()) && (time.Since(t.Time) > d)
```

Comment:
There's no unit test for this function.  The change seems good, but what would you thnk about adding a tiny, trivial test?